data/messages.proto

```diff
@@ -444,15 +444,23 @@ message LNURLAuth {
   bool jwt = 5;
 }
 
+message LNUrlPayImage {
+    string data_uri = 1;
+    string ext = 2;
+    bytes bytes = 3;
+}
+
 message LNUrlPayMetadata  {
-    repeated string entry = 1;
+    string description = 1;
```

Comment:
Doesn't this change create a problem in backward computability?
Since we serialize this into the payment item it seems that it won't be able to deserialize old items.

Reply:
🤔  I did not find any problem with my tests.

Do you have a test in mind that I can try to force this possible error?

Reply:
You need to use the old version to pay with lnurl and then upgrade to the new version.